kolibri/core/assets/src/api-resource.js

```diff
@@ -11,7 +11,7 @@ class Model {
    * @param {Resource} resource - object of the Resource class, specifies the urls and fetching
    * behaviour for the model.
    */
-  constructor(data, resource) {
+  constructor(data, resource, urlParams = {}) {
```

Comment:
Is this an unordered mapping object that corresponds to named match group patterns in an `api_urls.py` type file?